scripts/CorrModel.py

```diff
@@ -0,0 +1,63 @@
+from numpy import *
+from numpy import linalg as LA
+
+# Reduce correlation model
```

Comment:
Perhaps "Reduce" can be described better

Reply:
More description is given.